### ISSUE TITLE
Content Review Request - 686c Add/Remove Dependent (Stepchild Question)

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#62697]
## About your team

- Team name: Office of the CTO - Dependent Management Experience Team
- OCTO-DE product owner: Matt Self
- Product manager: Laura Steele
- Designer: Daniela Jones
- Accessibility specialist: none
- Content writer: Daniela Jones
- Slack channel: benefits-ce-dependents_exp
 
### Describe what you need in a few sentences:

We’re updating a form and want to confirm we’re doing it correctly.
- Adding a question and changing an existing question about stepchildren on the 686c Add/Remove Dependent Form
- We have questions about whether the word “parent” is sufficiently clear and would love feedback, or we can wait to address this in future research 

### Which are you doing?
- [ ] **Launching a new** page, tool, form, or app
- [x] **Making changes** to an existing page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [ ] Collaboration on content written by your team’s dedicated content writer 
- [x] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [ ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [x] Something else: We plan to bring this to the Benefits Portfolio Accessibility Office Hours.
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] Object-Oriented UX (OOUX) support
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe
Since November 2022, 686c claims involving stepchild information have been denied because all 686c forms submitted through lighthouse (VA.gov) are failing to mark the biological status of stepchildren, thus creating an incomplete form, leading to excessive denials of dependency claims.

### Have you worked with CAIA before? 
  - [ ] Yes
  - [x] No

### What's your team’s next step and its timing?

- **Our next step is** adjusting the form code based on the design recommendation. Technical discovery has already been completed.
- **Provide date if available**: Our current sprint ends on August 8th. We don’t have a specific deadline that we’ve committed to; the time constraint that we have is that claims are being denied due to this issue. 

#### Timing for your next step: 
  - [x] When content is ready
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [x] No

### If you're conducting research, when is that starting? 
**Provide date if available:** TBD, we will be conducting research more broadly on the dependent experience.

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
  - [ ] Yes
  - [x] No
 
**Note**: _If you checked yes, we’ll reach out to discuss details about the content in the react widget (we use these widgets to display entry points for new products to a certain percentage of users who visit our static pages)._

  
### When do you expect to launch your product to 100% of users?
 
*Please provide an estimated date so our team can create other relevant tickets.*
- **We expect to launch our product to 100% of users on:** TBD, as soon as content is ready and code changes have been implemented.
  
### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [x] No

## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- **[Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/60b84206484de1ab4604eb8ae5428317b448da87/products/view-and-update-dependents)**
- **User flows**: We don’t currently a high level process flow, but we do have a detailed form flow for the 686c in [Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1687976281975/2a9c6ca9ea6d955afa7977c777bbb72b15280903?sender=u6633b44b9a2da5d288ed7845)
- **[Prototype or mockups](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1690311086208/96d5f59b299912bc8c69542e6943d5b2213b9c72?sender=u6633b44b9a2da5d288ed7845)**
- **If this is in Sketch, which version or specific page should we focus on?**
- **Content source of truth**:
- **Research plan:**
- **Any epics/issues that may be helpful**: #61673 and #62106
- **Link to testing accounts** (please do not paste them in this ticket): Any staging account can be used to add stepchildren. Here is a link to the staging accounts: https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/mvi-staging-users.csv 

### Have you already worked with someone from OCTO leadership?
- [x] No
- [ ] Danielle
- [ ] Beth
- [ ] Mikki
- [ ] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [x] Let’s work async and meet as needed.

 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)

- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by steele-lm** [comment]
@juliepedtke @djones2113

**Comment by steele-lm** [comment]
@jstrothman

**Comment by kristinoletmuskat** [comment]
Hey! I confirmed that there's no major IA work here, so I'm going to untag myself. 


![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/122126772/946b2487-ee26-4e9b-9054-78cc65e88ef2)

**Comment by coforma-terry** [comment]
Hey @steele-lm - just letting you know this is in our queue for content feedback - thanks!

**Comment by laurwill** [comment]
Hi @steele-lm , 
I've added some comments to the mural with suggested wording changes. 

I also had 2 questions for you: 
- The SME guidance on the "date you married your stepchild's parent" question seems to be saying that we need to ask about 2 dates that might be different: the date of the marriage, and the date the stepchild "joined the household." Did you determine that the second date (date the child entered the household) isn't needed?
- Can Veterans add a stepchild as a dependent if they are no longer married to that child's parent? I think it's clearer to use "spouse" than "parent" in the questions about this person, but want to make sure we aren't confusing things for Veterans who have custody of stepchildren from former marriages.

**Comment by steele-lm** [comment]
@laurwill Thanks for the questions. I need to follow up with the SME on the "no longer married" question, but for the "date you married your stepchild's parent" the SME provided the following context:

_- The date the stepchild became a dependent would generally be considered the date in which the vet and the biological parent were married, as that is the date it would be assumed the spouse/child became financially dependent on the Veteran. VA however does not pay out additional dependency benefits unless the stepchild is also a member of the Veteran’s household, thus the need to know when the child became a member of the household. The child would be considered a member of the Veteran’s household if they live with the Veteran, or if they live apart due to medical reasons or one of them is incarcerated, attending school, or fulfilling a military service obligation. If the child became a member of the Vet’s household prior to the marriage of the parents, the earliest they could be added to the award is the date of marriage.  If they entered the household after the marriage, then the earliest is the date they entered the household._

**Comment by laurwill** [comment]
Thanks @steele-lm ! So then it sounds like we may need an additional question or two to determine when the child became part of the household, right? I don't see a question that captures that date in the current design.

**Comment by steele-lm** [comment]
@laurwill

-  The highest priority issue is adding the "Is this the biological child of your spouse?" field bc it's currently missing from the form. I recommend putting the marriage/household changes aside until we can conduct some additional research around the various different scenarios this form could encounter. **Should we update the Mural with the reduced scope?**
- Regarding divorces and stepchildren: Veterans remain entitled to additional benefits for a stepchild after divorcing the stepchild's biological or adoptive parent, as long as the veteran provides at least half of the stepchild's support.
- For determining when a child became a member of the household, the SME indicated that the date the stepchild became a dependent is generally considered the date in which the Veteran and the biological parent were married bc it's assumed that is when the spouse/child became financially dependent on the Veteran. If we want to avoid asking about marriage dates or multiple dates, we also have the option of leveraging the existing form language that asks for the "Date stepchild became your dependent".

**Comment by steele-lm** [comment]
Scope decreased to only include:

- Add question: "Is this child the biological child of your spouse?"
- Edit existing question: Change from "Date stepchild became your dependent" to "Date the child entered your household"

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/134541644/569c00ab-6138-4390-8ab3-2cda95e88afb)
